README.md

```diff
@@ -234,7 +234,71 @@ Templates in React are writtien inside the JavaScript file using the [JSX langua
 
 # Inputs and Outputs
 
-> TODO
+### Angular
+
+#### Parent Component
+```js
+@Component({
+  selector: 'parent',
+  template: `
+    <child
+      [user]="user"
+      (onEdit)="editedUser($event)"
+    >
+
+    </child>
+  `
+})
+
+export class ParentComponent {
```

Comment:
We should give these more meaningful names, like `Settings` and the child `UserPreview` (the same goes with selectors).